proto/viam/service/slam/v1/slam.proto

```diff
@@ -45,6 +45,13 @@ service SLAMService {
     };
   }
 
+  // GetProperties returns properties, such as mapping_mode, of the current slam service 
```

Comment:
Add comments for the mapping mode types here and above the enum! Well done adding a comment to the slam service :D